modules/nbde-rekeying-all-nbde-nodes.adoc

```diff
@@ -42,7 +42,7 @@ spec:
     spec:
       containers:
       - name: tang-rekey
-        image: registry.access.redhat.com/ubi8/ubi-minimal:8.4
+        image: registry.access.redhat.com/ubi9/ubi-minimal:8.4
```

Comment:
I don't think there is a `registry.access.redhat.com/ubi9/ubi-minimal:8.4`tag 

Maybe replacing `8.4` with `latest` ?

Reply:
I updated to `ubi9/latest` where possible and changed the shas to `@sha256:123...` to indicate the string output.